lib/internal/test_runner/tests_stream.js

```diff
@@ -27,6 +27,10 @@ class TestsStream extends Readable {
     }
   }
 
+  begin(nesting, file, testNumber, name, details, directive) {
```

Comment:
does anything call this method?
I would start by adding this method to this test file, then make sure it passes https://github.com/nodejs/node/blob/1b177932a124c1f9cc10a3239b62a07d642169d3/test/parallel/test-runner-reporters.js#L93

Reply:
no this is the new event so, not called anywhere yet.